neuvector-azure-keyvault/Chart.yaml

```
@@ -1,5 +1,5 @@
 apiVersion: v1
 appVersion: "1.0"
 description: A Helm chart for NeuVector
-name: chart-neuvector
-version: 0.0.1
+name: neuvector-azure-keyvault
```

maybe? it's not hugely related to azure-keyvault, it's just applying our automation that isn't available in the configmap?
```suggestion
name: neuvector-automation
```

---

The main point there for me is that it could be integrated in the official Neuvector chart if it didn't use key vault to get its secrets. That is why I chose that name. If it was only for automation there would not be a separate chart in the first place